products/sle/main.pm

```diff
@@ -810,11 +810,13 @@ elsif (get_var("VIRT_AUTOTEST")) {
             loadtest "virt_autotest/virsh_external_snapshot";
         }
     }
-    elsif (get_var("ENABLE_SRIOV_NETWORK_CARD_PCI_PASSSHTROUGH")) {
+    #those tests which test extended features, such as hotpluggin, virtual network and SRIOV passhthrough etc.
+    #they can be seperated from prj1 if needed
+    elsif (get_var("VIRT_EXTENDED_TEST")) {
```

Comment:
From this parameter name, it is not easy to clearly tell where tests with this setting locate in the chain. In long term consideration, if in future more than 2 layers of tests are chained, can you figure out some parameters to differentiate these tests so that different blocks of loadtest can be applied?

Reply:
VIRT_EXTENDED_TEST can include any tests which with existing guest. For what test to load,  specific settings will control, such as 
loadtest "virt_autotest/sriov_network_card_pci_passthrough" if get_var("ENABLE_SRIOV_NETWORK_CARD_PCI_PASSSHTROUGH");
loadtest "virtualization/universal/hotplugging"             if get_var("ENABLE_HOTPLUGGING");
loadtest "virtualization/universal/storage"                 if get_var("ENABLE_STORAGE");

ie. you can create a testsuite with VIRT_EXTENDED_TEST by grouping some ENABE_xxx settings.

Reply:
Do you mean something like POST_PRJ1_TEST?